Pod/Classes/SKYRecord.h

```diff
@@ -41,16 +41,16 @@ extern NSString *const SKYRecordTypeUserRecord;
 /// Undocumented
 + (instancetype)recordWithRecordID:(SKYRecordID *)recordId data:(NSDictionary *_Nullable)data;
 
+/// Undocumented
+- (instancetype)init NS_UNAVAILABLE;
 /**
  Instantiates an instance of <SKYRecord> with the specified record type with a randomly generated
  <SKYRecordID>.
 
  @param recordType Record type of the record.
  @return An instance of SKYRecord.
  */
-- (instancetype)init NS_UNAVAILABLE;
-/// Undocumented
-- (instancetype)initWithRecordType:(NSString *)recordType;
+ - (instancetype)initWithRecordType:(NSString *)recordType;
```

Comment:
hi @bicep thanks for your PR! There is an extra space at the beginning of this line and you should remove that : ) sorry the clang check is a little unforgiving.

I could also change that for you but I guess you might want a chance to fix it yourselves so it merges cleanly.